src/forwarder.ts

```diff
@@ -49,6 +49,14 @@ function isLogGroup(value: any): value is LogGroupResource {
  * @param functionArn The forwarder ARN to be validated
  */
 async function validateForwarderArn(aws: Aws, functionArn: CloudFormationObjectArn | string) {
+  if (functionArn === "arn:aws:lambda:us-east-1:000000000000:function:datadog-forwarder") {
```

Comment:
From the perspective of the code calling `validateForwarderArn`, this is a hidden case which breaks the expectation of what the function does. It's better to move the logic to skip validation to where 'validateForwarderArn' is called instead. The other issue is the arn is a magic value. An alternative might be to add another config value, eg `validateForwarder`,  default it to true and check it before calling `validateForwarderArn`. Then we set it to false for the integration tests.

Reply:
I introduced a validateForwarder parameter, let me know how that looks